contents/blog/running-group-targeted-ab-tests.md

```diff
@@ -0,0 +1,93 @@
+---
+title: When and how to run group-targeted A/B tests
+sidebar: Docs
+showTitle: true
+author: ['lior-neu-ner']
+date: 2023-04-24
+featuredImage: ../images/blog/running-group-targeted-ab-tests/featured-image.png
```

Comment:
White might be a little jarring as a background colour, maybe try the same color as the tutorial backgrounds? (E5E7E0)

Reply:
think I'll just ask Lottie for an image. Fits in better with the other blogs as well

Reply:
Assuming this stays in the tutorials section, it should retain the same style. I'm sure there's one we can reuse.

Reply:
Ah, I see it's moved to the blog already. That being the case, @Lior539 you need to update the frontmatter to the blog version. You can grab it from any existing blog post.